svc-api/handle/systems.go

```diff
@@ -312,3 +313,10 @@ func (sys *SystemRPCs) ChangeBootOrderSettings(ctx iris.Context) {
 	ctx.StatusCode(int(resp.StatusCode))
 	ctx.Write(resp.Body)
 }
+
+// CreateVolume is the handler to create a volume under storage
+// from iris context will get the request and check sessiontoken
+// and do rpc call and send response back
+func (sys *SystemRPCs) CreateVolume(ctx iris.Context) {
+	// This function yet to be implemented
```

Comment:
retrun StatusNotImplemented status   after code is added remove that

Reply:
Addressed the comment